src/olympia/amo/templates/amo/fake_fxa_authorization.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h2> {{ _('Register or Log in') }} </h2>
+  <form id="fake_fxa_authorization" method="get" action="{{ url('auth:accounts.authenticate') }}">
+      {% for key, value in request.GET.items() %}
+        <input type="hidden" name="{{ key }}" value="{{ value }}" />
+      {% endfor %}
+      <input type="hidden" name="code" value="fakecode" />
```

Comment:
what's `fakecode` for?

Reply:
The authenticate callback view checks that a `code` is passed, we never enter the function that uses its value but it needs to be present in the query for it to work. I didn't want to add yet another check to bypass that.